.travis.yml

```diff
@@ -0,0 +1,22 @@
+notifications:
+  slack:
+    secure: j3t/N/O0YD1BZsiihdrw5yYLFtyYZgVPkxcCIr0yWEYRhk1OWw2RS/UcM+fjQjaRZMhSn+k6vFW5YXNMEPEDvVbuTt75Sj9Gy3YeSYmO6b9ZIUBDDkamBMMx1c2YUxZTZaL7gRaLF8H9+xz9MIMvsGeJAjSs0H8QtOMscWEO6n+Uko/HX/Sw/p/EpdnWBj6CBu+WKutmRM/4QVRVe+kJirLrx8Hux7ejCnxskdfbh/i/beq6Oy9PQEsBLLn2+kGWxXb2lurCdH/eQtnshIJLHO5ee0XDEyp+H1QFXKQnu//QIAAW5QcEU6hM0aP1j37n5J/uRsARf86e6iYSIvP3esvuKGBLnb9iqR/qx/aekqc8fIXmi8a5Slgor4m2W2RxHlTKijdwwzIzX1BFOV3hxJ3ZmeOs8j95OpesQfHS4tAxhbLowJvZT38pBm3rOwTWeotz1CE81Zs2ycnjVI93a5xG9VteXh4xOfrHgBqkOgaohrl9wQknc7x0W5xGe1ycf6o6pLln6nFs9MqgXjFH7XTOajQL+mChukA7iRFznnE97xMN4rrKiRzluZ4C8A6z0GXWyYuI2FQdg5mj8zh2S4NkuJvud3vHanCDHjVkRDVv5AIi7qMGgt2c1Cv/lqGZhwT1nXpSpgF5TsOtKOHfa54wzqnJiEucWacuwdQziHc=
+  email:
+    recipients:
+    - mark.mcewen@capita.com
+    on_success: never
+    on_failure: always
+services:
+- docker
+script:
+- docker build --target test --tag todo-app:test .
+- docker run -e SECRET_KEY=$SECRET_KEY -e TRELLO_KEY=$TRELLO_KEY -e TRELLO_TOKEN=$TRELLO_TOKEN
+  -e TRELLO_BOARD_ID=$TRELLO_BOARD_ID -p 5000:5000 --mount type=bind,source="$(pwd)"/todo_app,target=/project/todo_app
```

Comment:
Given that you copy all your test files in directly, it might also make sense to copy your code in, since the bind mount here won't achieve much (and means users can only run the tests from this image if they also have access to the code elsewhere)
Worth also noting that there's no need to expose the port (since the tests are running inside the docker container as well as the code)